packages/driver/src/cy/commands/navigation.ts

```diff
@@ -315,6 +315,11 @@ const stabilityChanged = async (Cypress, state, config, stable) => {
   const onPageLoadErr = (err) => {
     state('onPageLoadErr', null)
 
+    // If the error thrown is a cypress error, return it instead of throwing a cross origin error.
+    if ($errUtils.isCypressErr(err)) {
```

Comment:
I wonder if this will fix the error I've been seeing in my branch! Hoping it will 😄